### DISCUSSION TITLE
glTF Working Group Meeting 2022-02-03

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/gltf-extensions#55] [General]
# OMI glTF Working Group Meeting 2/03/2022

This meeting will be on 2/03/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- OMI_audio_emitter check-in
- Collider extension demo??? 👀 
- Avatar Animation Retargeting

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by robertlong** [comment]
Avatar Animation Retargeting
- https://github.com/godotengine/godot/pull/56902
- Question: Should we reduce the scope of this standard to retargeting animations from a single skeleton type to a model with the same skeleton type or should we support retargeting across skeleton types?

Collider
- https://blog.mauve.moe/slides/omi-gltf-colliders-intro/#1
- Compound colliders on children or array?
- What transform constraints should we impose for compound colliders or colliders in general?
- Where should trigger property go?
- We should come to the next meeting with examples of collider / physics / trigger situations and how we might define them using the spec.

Audio Emitter
- We decided as a group to merge https://github.com/omigroup/gltf-extensions/pull/46
- We also merged the audio emitter draft into main
- Waiting on others to add themselves as contributors before we hold a vote to move to stage 2 next week.
